packages/wxt/src/builtin-modules/unimport.ts

```diff
@@ -147,4 +148,26 @@ export default {
 };
 `,
   };
+
+  const javaScriptFileDirname = path.dirname(options.eslintrc.filePath);
+  const javaScriptFileExtension = path.extname(options.eslintrc.filePath);
+  const javaScriptFileBasename = path.basename(
+    options.eslintrc.filePath,
+    javaScriptFileExtension,
+  );
+
+  const typeScriptFilePath = path.join(
+    javaScriptFileDirname,
+    `${javaScriptFileBasename}.d.ts`,
+  );
```

Comment:
This could be simplified a little bit

```suggestion
  const typeScriptFilePath = options.eslintrc.filePath.slice(0, -extname(options.eslintrc.filePath)) + ".d.ts"
```

The second slice argument might be off by one, I can never remember if `extname` returns the extension with or without the dot. Or just remove the first dot from `".d.ts"`.